main.go

```diff
@@ -1,4 +1,5 @@
-//+build !test
+//go:build !test
```

Comment:
This syntax is for go1.17 and later. 

Given that acme-dns still claims to compile with go1.13, the original syntax should be maintained.

Reply:
this was done automaticlly by the IDE/go tools, i think the compatiblity should be ok since line 2 has the old syntax. I think its good to have both syntax supported since the old one may gets removed one day.